src/test/java/EditReceiptServletTest.java

```diff
@@ -135,7 +135,7 @@ public void tearDown() {
   }
 
   @Test
-  public void doPostEditsReceipt() throws IOException {
+  public void doPost_editsReceipt() throws IOException {
```

Comment:
Are you allowed to omit the situation being tested?

Reply:
From #109:
> Example:
> transferMoney_deductsFromSource
> transferMoney_addsToDestination
> transferMoney_insufficientFunds_throwsException
> multiply_twoNegativeValues_returnsPositiveValue
> multiply_anythingWithNan_returnsNan
> inputIsValid_conversionSucceeds

I assume yes because some of the examples omit it.